app/src/main/java/ch/epfl/qedit/model/users/Participant.java

```diff
@@ -0,0 +1,41 @@
+package ch.epfl.qedit.model.users;
+
+import java.util.Date;
+
+import ch.epfl.qedit.model.Quiz;
+
+public class Participant extends User {
+    private Quiz quiz;
+    private Date startTime;
+
+
+    public Participant(String firstName, String lastName, int userId, String language) {
+        super(firstName, lastName, userId, language);
+    }
+
+    //allows participant to select a quiz or be assigned one
+    public boolean getQuiz(Quiz quiz){
```

Comment:
Do we really need the methods `setQuizStartTime`, `createQuiz`, `editQuiz`, `getQuiz`, .... ?
I feel like it creates a lot of coupling between `User` and `Quiz`, which are conceptually not so related. IMO, it will make the code harder to test and maintain.
Furthermore, I think that the "model" part of the MVVM shouldn't care too much about behavior, but rather just hold the data.
What do you think ?

Reply:
As it is still unclear to me how the editor and the quiz will interact I added these methods as "might be useful later" we'll have to talk about these interactions. Other than that, I agree with the fact that the "model" shouldn't care too much about behavior.